ivy/functional/frontends/__init__.py

```diff
@@ -3,3 +3,87 @@
 from . import jax
 from . import torch
 from . import tensorflow
+import importlib
+
+latest_version = {"torch": "1.12", "tensorflow":"2.9.0", "numpy":"1.23.2", "jax":"0.3.16"}
+
+
+def version_extractor(name, version):
```

Comment:
we should make this function name more informative, such as `fn_name_from_version_specific_fn_name`.